Re-submit of earlier PR combined into one commit

SkorpionNL opened the pull request:
- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.
- Add check for reachable state for sensors.
- Added On/Off plug-in unit as another Smart Plug type
- New Smart plug Icon
- Fixed some typo's
- Added support for:
Aqara Wireless Button (WXKG11LM)
Ikea Symfonisk Sound Controller

SkorpionNL committed:
Initial commit

Added On/Off plug-in unit as another Smart Plug type
New Smart plug Icon
Fixed some typo's
Added support for:
- Aqara Wireless Button
- Ikea Symfonisk Sound Controller

SkorpionNL committed:
Requested changes from PR master branch

- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.

SkorpionNL committed:
Revert "Requested changes from PR master branch"

This reverts commit 425273483c3fffdc4d8d97607bce81a16cbeb9bc.

SkorpionNL committed:
Requested changes from PR master branch

- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.
- Add check for reachable state for sensors.

SkorpionNL committed:
Merge branch 'master' of https://github.com/SkorpionNL/HomeyCONZ

SkorpionNL committed:
Several additions into one commit

- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.
- Add check for reachable state for sensors.
- Added On/Off plug-in unit as another Smart Plug type
- New Smart plug Icon
- Fixed some typo's
- Added support for:
Aqara Wireless Button (WXKG11LM)
Ikea Symfonisk Sound Controller

SkorpionNL commented:
Some things went wrong when I installed GitHub Desktop today, so I re-made my master branch and re-submitted my PR including the changes you requested on my previous PR.

SkorpionNL committed:
Delete nbhttp.js

Moved to node_modules folder

notabene00 commented on app.json:
```diff
@@ -619,58 +675,90 @@
+        "id": "start_ccw_rotate",
         "title": {
-          "en": "Button held"
+          "en": "Button has been rotated counter clockwise",
```
Can you please change "button" to "device" or something (with translation to nl)
So I can use this trigger in tradfri wireless (round) dimmer implementation, which is not "button"

SkorpionNL replied:
Is "remote" ok?

notabene00 replied:
Yes

SkorpionNL replied:
Done

notabene00 commented on app.json:
```diff
@@ -619,58 +675,90 @@
+        "id": "start_cw_rotate",
         "title": {
-          "en": "Button released"
+          "en": "Button has been rotated clockwise",
```
Same here

SkorpionNL committed:
Update app.json

Changed Button into Remote and Dutch translations accordingly for Symfonisk-remote driver id.

fantomnotabene commented:
Great! Thanks!
